bito-action-script/bito-actions.sh

```diff
@@ -36,9 +36,9 @@ do
       agent_instance_secret="${arg#*=}"
       agent_instance_secret="${agent_instance_secret//\"}"
       ;;
-    git_url=*)
-      git_url="${arg#*=}"
-      git_url="${git_url//\"}"
+    pr_url=*)
+      pr_url="${arg#*=}"
+      pr_url="${pr_url//\"}"
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/gitbito/CodeReviewAgent/pull/33#issuecomment-2278244118>#e844b6</a><span> - 08/09/2024, 03:48 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The script lacks input validation for the newly renamed 'pr_url' parameter.
</div>
<div id="fix">
<b>Fix:</b> Add input validation for the 'pr_url' parameter to ensure it's not empty and follows the expected format (e.g., a valid URL).
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -39,3 +39,7 @@
  pr_url=*)
    pr_url="${arg#*=}"
    pr_url="${pr_url//\"}"
 +  if [[ -z "$pr_url" || ! "$pr_url" =~ ^https?:// ]]; then
 +    echo "Error: Invalid or empty PR URL provided"
 +    exit 1
 +  fi
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
Validation of script is added later part of the script.